corehq/apps/saved_reports/models.py

```diff
@@ -702,3 +704,15 @@ def update_attributes(self, items):
     def verify_start_date(self, start_date):
         if start_date != self.start_date and start_date < datetime.today().date():
             raise ValidationError("You can not specify a start date in the past.")
+
+
+class ScheduledReportsCheckpoint(models.Model):
```

Comment:
could do with a docstring to outline it's purpose

Reply:
Thanks, will add